packages/actions/src/ActionHelpers.ts

```diff
@@ -152,6 +152,9 @@ function getImageVariableValue(variableName: string | Variable): string {
 /**
  * Sets the value of a variable by its name or variable object.
  *
+ * If the value is set on a ShortTextVariable, it cannot contain any type of
```

Comment:
This might be more explicit
```suggestion
 * If the value is set on a ShortTextVariable, it must not contain any type of
```